examples/other_languages.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import os
+from subprocess import PIPE, Popen
+
+# insert your client_token into sigopt_creds.py
+# otherwise you'll see "This endpoint requires an authenticated user" errors
+from sigopt_creds import client_token
```

Comment:
Can you change all the examples to be command line args then? editing this
sigopt_creds file is usually just annoying

On Fri, Feb 12, 2016 at 4:14 PM, Alexandra L Johnson <
notifications@github.com> wrote:

> In examples/other_languages.py
> <https://github.com/sigopt/sigopt-python/pull/29#discussion_r52815889>:
>
> > @@ -0,0 +1,57 @@
> > +import argparse
> > +import os
> > +from subprocess import PIPE, Popen
> > +
> > +# insert your client_token into sigopt_creds.py
> > +# otherwise you'll see "This endpoint requires an authenticated user" errors
> > +from sigopt_creds import client_token
>
> Trying to keep all of the examples consistent
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sigopt/sigopt-python/pull/29/files#r52815889>.
>